Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -2101,10 +2101,14 @@ inline void BuilderCustom::applyValueContainIntrinsicWidth(BuilderState& builder
     auto& list = downcast<CSSValueList>(value);
     ASSERT(list.length() == 2);
```

Comment:
This can go away if you use CSSValuePair.

Reply:
Nice, fixed.